package.json

```diff
@@ -203,6 +203,7 @@
     "distclean": "npm run -s clean && npm run -s rm -- node_modules",
     "docker": "docker run -it --name wp-calypso --rm -p 80:3000 wp-calypso",
     "eslint-branch": "node bin/eslint-branch.js",
+    "gutenblock": "node bin/create-scripts/block.js",
```

Comment:
do you think we could stick with the `build block` nomenclature? I think it would be reasonable especially since this isn't specific to Gutenberg or blocks

Reply:
Sure, good point. Updated it to `build-block` in cdc418ec35c309c5a7266bb9f4a68057fb10259d.

Reply:
+1 for `build-block` — We should make sure not to use the term "Gutenberg" too much in scripts and APIs.

`build-` prefix fits nicely with other build scripts `build-css/server/client/devdocs/docker...`